src/serac/physics/nonlinear_solid.hpp

```diff
@@ -121,6 +121,13 @@ class NonlinearSolid : public BasePhysics {
   void setTractionBCs(const std::set<int>& trac_bdr, std::shared_ptr<mfem::VectorCoefficient> trac_bdr_coef,
                       int component = -1);
 
+  /**
+   * @brief Set the external force vectors on the domain
+   *
+   * @param[in] ext_force_coefs The sd::vector of vector-valued external force coefficients applied to the domain
+   */
+  void setExternalForces(std::vector<std::shared_ptr<mfem::VectorCoefficient>> ext_force_coefs);
```

Comment:
I think `setBodyForces` is a bit clearer for what this is actually doing.

Reply:
Also, I think having this with a single coefficient and then pushing them to a vector internally would be easier to use in the common gravity load case. This is similar to how we handle the boundary conditions.

Reply:
Hmm. I just wanted to clarify if you think we should internally have vectorcoefficient internally, or to take as input only one vectorcoefficient.

I was envisioning the case of two loads (gravity + applied torque) and how we might want to accommodate that. In other codes, the gravity would have a density term and that might be modified, where the applied torque might not. Someone else would then have to combine them, whereas if they are store separately, they can be mutated individually and we can combine them in any way we like.

I do see that there would be a performance benefit of internally having one vector coefficient since we wouldn't have to loop over multiple domain integrators.

Reply:
Oh, input as only one vector coefficient. So internally, it's is still a `std::vector<VectorCoefficient>`, but the function call sets them one at a time. This means to have multiple body loads, a user would do this
```
solid.setBodyForce(force1);
solid.setBodyForce(force2);
```
instead of constructing a `std::vector` themselves.

Reply:
I wouldn't worry too much about the performance hit as it will probably change significantly with the upcoming `VariationalForm` anyway.

Reply:
maybe `addBodyForce` for clarity.